### ISSUE TITLE
`module!`: `author` should take a list

### ISSUE THREAD

**Issue by ojeda** [Rust-for-Linux/linux#244]
`MODULE_AUTHOR` is supposed to be repeated in C.

Perhaps we should rename it `authors`, too.

**Comment by wcampbell0x2a** [comment]
Is this the API you are looking for?
```
-    author: b"Rust for Linux Contributors",
+    author: [b"Rust for Linux Contributors", b"john@doe"],
```

**Comment by ojeda** [comment]
Something like that, yeah!

**Comment by GuilhermeGiacomoSimoes** [comment]
@ojeda 
I sent a patch for this 

https://lore.kernel.org/all/20241206192244.443486-1-trintaeoitogc@gmail.com/

**Comment by ojeda** [comment]
Thanks!

@wcampbell0x2a submitted a PR a while ago (https://github.com/Rust-for-Linux/linux/pull/904) that was reviewed etc., did you see that? I think the patches were never submitted, though.

**Comment by GuilhermeGiacomoSimoes** [comment]
@ojeda 

Yes, I saw that, but since @wcampbell0x2a never sent his patches, and it was a long time ago, I did the change my way and sent my patches.

**Comment by ojeda** [comment]
In general, I recommend contacting the person, even if just out of deference. If there is no reply, then you could still consider submitting their patches etc.

Moreover, it is also a good idea to mention prior work in the commit message, and discuss why you used a different approach, since the PR was discussed by a few people back then.

Thanks!

**Comment by ojeda** [comment]
https://lore.kernel.org/rust-for-linux/20250309175712.845622-1-trintaeoitogc@gmail.com/

Applied to `rust-next`, so closing since it will likely land in mainline.
